movie_library/models/actor.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+#from .movie import Movie
+
+
+class Actor(models.Model):
```

Comment:
you should import from BaseModel and add created and updated filed to all models